scicamera/previews/null_preview.py

```diff
@@ -9,24 +8,22 @@
 class NullPreview:
     """Null Preview"""
 
-    def thread_func(self, camera):
+    def thread_func(self):
         """Thread function
 
         :param camera: Camera object
         :type camera: Camera
         """
-        sel = selectors.DefaultSelector()
-        sel.register(camera.notifyme_r, selectors.EVENT_READ, self.handle_request)
-        self._started.set()
-
-        # TODO(meawoppl) - abort flag and select can be polled in parallel
-        # which will make startup/shutdown faster
         while not self._abort.is_set():
-            events = sel.select(0.2)
```

Comment:
Previously mode changes + start/stop calls would take an average of 0.1 extra seconds...